store/tikv/tikvrpc/tikvrpc.go

```diff
@@ -389,6 +389,13 @@ func (req *Request) TxnHeartBeat() *kvrpcpb.TxnHeartBeatRequest {
 	return req.Req.(*kvrpcpb.TxnHeartBeatRequest)
 }
 
+// EnableStaleRead enable the stale read
+func (req *Request) EnableStaleRead() {
+	req.StaleRead = true
+	req.ReplicaReadType = kv.ReplicaReadMixed
```

Comment:
Now that `req.ReplicaRead` is false, why set `req.ReplicaReadType`?

Reply:
`ReplicaReadType` is used for picking `store` in `GetTiKVRPCContext`.